assets/src/components/amp-support/error-sources.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * To render error source information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Error data.
+ * @return {JSX.Element|null} HTML markup for error source data.
+ */
+export function ErrorSources( { data: errorSources } ) {
+	if ( 'object' !== typeof errorSources ) {
```

Comment:
Since it is meant to be an array, the check should be more specific:
```suggestion
	if ( Array.isArray( errors ) ) {
```